controllers/cert.go

```diff
@@ -39,7 +39,15 @@ func (c *ApiController) GetCerts() {
 	sortOrder := c.Input().Get("sortOrder")
 
 	if limit == "" || page == "" {
-		maskedCerts, err := object.GetMaskedCerts(object.GetCerts(owner))
+		var (
+			maskedCerts []*object.Cert
+			err         error
+		)
+		if field == "scope" && value != "" {
+			maskedCerts, err = object.GetMaskedCerts(object.GetCertsByScope(owner, value))
```

Comment:
and we can use GetPaginationCerts instead creating GetCertsByScope

Reply:
fixed